e2e/pages/program-page.ts

```diff
@@ -4,7 +4,8 @@ export class ProgramsPage {
   constructor(readonly page: Page) {}
 
   readonly programsTable = () => this.page.getByRole('table', { name: /program enrollments/i });
-  readonly editProgramButton = () => this.page.getByRole('button', { name: /edit program/i });
+  readonly overflowButton = () => this.page.getByRole('button', { name: 'Options' });
+  readonly editProgramButton = () => this.page.getByRole('menuitem', { name: 'Edit' });
```

Comment:
Minor nitpick, it would be great if we could stick to using the regex of matching case instead of strict typing it.
```suggestion
  readonly overflowButton = () => this.page.getByRole('button', { name: /options/i });
  readonly editProgramButton = () => this.page.getByRole('menuitem', { name: /edit/i });
```